docs/library.md

```diff
@@ -64,6 +64,7 @@ Arguments include:
   - `enableCors`: Enables some generous CORS settings for the GraphQL endpoint. There are some costs associated when enabling this, if at all possible try to put your API behind a reverse proxy.
   - `exportJsonSchemaPath`: Enables saving the detected schema, in JSON format, to the given location. The directories must exist already, if the file exists it will be overwritten.
   - `exportGqlSchemaPath`: Enables saving the detected schema, in GraphQL schema format, to the given location. The directories must exist already, if the file exists it will be overwritten.
+  - `sizeLimit`: Set the maximum size of JSON bodies that can be parsed (default 100kB).
```

Comment:
Could we change this to `bodySizeLimit` and add a similar note about pretty bytes?